docs/sources/alerting/fundamentals/notifications/index.md

```diff
@@ -11,19 +12,35 @@ keywords:
 
-You can also use message templating to customize notification messages for contact point types.
+## Notification policies
 
-## Supported contact point types
+Notification policies control when and where notifications are sent. A notification policy can choose to send all alerts together in the same notification, send alerts in grouped notifications based on a set of labels, or send alerts as separate notifications. Each notification policy can have its own timings configuration that control how often notifications should be sent; and can reference one or more mute timings to inhibit notifications at certain times of the day and on certain days of the week.
 
-The following table lists the contact point types supported by Grafana.
+Notification policies are organized in a tree structure where at the root of the tree there is a notification policy called the root policy. There can be only one root policy, and the root policy cannot be deleted. The root policy also happens to be where all firing and resolved alerts are sent.
+
+Specific routing policies are children of the root policy, and can be used to match either all alerts, or a subset of alerts based on a set of labels. An example of a specific routing policy could be sending high priority alerts to Pagerduty and sending low priority alerts as emails. Another example of a specific routing policy could be sending infrastructure alerts to the Ops team, and all other alerts to another team.
+
+When an alert is sent to the root policy it looks at each specific routing policy and sends the alert to the first specific routing policy that matches the alert. If there are no specific routing policies, or none which match the alert, then the root policy is the matching policy.
+
+More information on how to configure notification policies can be found [here]({{< relref "../../manage-notifications/create-notification-policy/" >}}).
+
+## Contact points
+
+Contact points contain the configuration for sending notifications. A contact point is a list of integrations each of which sends a notification to a particular email address, service or URL. Contact points can have multiple integrations of the same kind, or a combination of integrations of different kinds. For example, a contact point could contain a Pagerduty integration; an email and Slack integration; or a Pagerduty integration, a Slack integration, and two email integrations. A contact point can also be configured with no integrations in which case no notifications are sent.
+
+However a contact point cannot send notifications until it has been added to a notification policy. A notification policy can only send alerts to one contact point, but a contact point can be added to a number of notification policies at the same time. When an alert matches a notification policy, the alert is sent to the contact point in that notification policy, which then sends a notification to each integration in its configuration.
```

Comment:
I'm hesitant to add this suggestion as we haven't used the word entry before to talk about an individual notification policy, and I fear it might confuse the reader. What do you think?

Reply:
I thought this too, any alternate wording is fine IMO, I just felt the original of:

> A notification policy can only send alerts to one contact point

Is not necessarily true since a policy can have children.

Reply:
Even when it matches a child policy doesn't only the child policy contact point receive the alert? Or are alerts sent to the contact points of all matching notification policies?

Reply:
With Continue=true, you can make one policy node appear to dispatch to multiple points, but thats an advanced feature. 🤷 

I think this also suffers from conflating the term here. Does a "notification policy" mean the whole tree? A subtree? A single node? Etc.